docs/sop/PRODUCTION_TROUBLESHOOT.md

```diff
@@ -279,4 +279,58 @@ The terraform inventory script would look into the terraform state and should be
 $ ../../../scripts/ansible_init.sh --env live
 ```
 
+### How to create a new user account in the bastion server
 
+If a user wants to execute tools, eg. readme generate, excel spreadsheet uploader, etc in the bastion server, a user account with appropriate permission has to be created first.
+The creation can be achieved as below:
+```
+% cd /gigadb-website/ops/infrastructure/envs/live
+# make sure bastion server is up and running
+% terraform output
+# execute the user playbook to create a user account
+% ansible-playbook -i ../../inventories users_playbook.yml -e "newuser=$user" --extra-vars="gigadb_env=live"
+# check if bastion_public_key_$user (live) exists in the upstream gitlab variables page
+# the private key will be saved as /output/privkeys-$bastion-ip/$user
+# update the permission of the private key
+% chmod 500 output/privkeys-$bastion-ip/$user
+# connect to the bastion server
+% ssh -i output/privkeys-$bastion-ip/$user $user@$bastion-ip
+```
+
+### What if a user has accidentally deleted (or corrupted) their .ssh/authorized_keys
+
+Please contact tech team if a user suspects that the /home/$user/.ssh/authorized_keys has accidentally deleted or corrupted, tech team can help to retrieve it from the [upstream gitlab variable page](https://gitlab.com/gigascience/upstream/gigadb-website/-/settings/ci_cd) after logging in 
+and put it back to the bastion server as /home/$user/.ssh/authorized_keys, while the authorized keys in gitlab would be in the form of bastion_public_key_$user. 
+
+### What if a user has lost their private keys (or they are compromised)
+
+Please contact tech team if a user has lost their private keys, tech team will first delete the existing /home/$user/.ssh/authorized_keys in the bastion server and also in the [upstream gitlab variable page](https://gitlab.com/gigascience/upstream/gigadb-website/-/settings/ci_cd), 
+then execute the `user_playbook` for the user:
+```
+% ansible-playbook -i ../../inventories users_playbook.yml -e "newuser=$user" --extra-vars="gigadb_env=live"
+```
+which will generate a new pair of ssh keys, the new private key will then be sent to the user, while the public key will be pushed to the the gitlab variable page and also the bastion server.
+
+### What if a user still cannot connect to the bastion server even they have a valid private key file in their computer and the corresponding public key in the baston server
+
+This may be because the sshd service in the bastion server has not been started properly, tech team will:
+```
+# login bastion server as a centos user
+% ssh -i /path/to/id-rsa-aws-hk-gigadb.pem centos@$bastion-ip
+# restart the sshd service
+systemctl restart sshd.service
+```
+
+### How to connect to the bastion server for a Windows user
```

Comment:
Hi @kencho51, 
I probably should have been clearer, by users on windows, I meant what to do if If we need to create a bastion account for users that use Windows (like Yannan and ChrisH for now). As you've figured out, most Windows user use Putty. The problem with Putty is that it doesn't support the PEM format. It only supports the PPK format. So when creating a bastion user to be used by a Putty user, there is an extra step to do in order to convert the PEM file into PPK:
```
$ puttygen lily.pem -o lily.ppk
```
It requires prior installation of putty on macOS: 
```
$ brew install putty
```
That's the info that I meant to be documented (although we can keep the question/answer you've added as well in case we need to know that too)

Reply:
Hi @rija,

The step for format conversion has been included.